package.json

```diff
@@ -158,7 +159,7 @@
     "@uniswap/router-sdk": "^1.3.0",
     "@uniswap/sdk-core": "^3.2.2",
     "@uniswap/smart-order-router": "^3.6.1",
-    "@uniswap/token-lists": "^1.0.0-beta.30",
+    "@uniswap/token-lists": "^1.0.0-beta.31",
```

Comment:
does this require an upgrade in the same PR or is it unrelated

Reply:
on second thought I can separate this change out into another PR. this PR can just be moving the same logic over. ill followup with a PR that upgrade the version to address the CG list issue.